src/serializer/ResidualHeapSerializer.js

```diff
@@ -659,10 +659,11 @@ export class ResidualHeapSerializer {
     let name;
     if (val instanceof FunctionValue) name = this._getFunctionName(val);
     else name = this.valueNameGenerator.generate(val.__originalName || "");
```

Comment:
The `name` used to mean something, but after you change, it doesn't, and yet it's used in debugging output further below.